.env.j2

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Your app isn't working yet. 

The aim is that you can visit `http://52.56.48.179:5000` in your browser (the host IP address followed by port 5000).

But you're very close. I believe the only issue is this file. This doesn't generate the same file as your .env file. You need these lines for it to look the same as your .env.template or .env or .env.test files. 
```
FLASK_APP=todo_app/app
FLASK_ENV=development
SECRET_KEY=secret-key
```

FLASK_ENV isn't required for the app to run, but the other two are.